src/wheel/bdist_wheel.py

```diff
@@ -202,12 +281,8 @@ def get_tag(self):
             else:
                 abi_tag = str(get_abi_tag()).lower()
             tag = (impl, abi_tag, plat_name)
-            supported_tags = pep425tags.get_supported(
-                self.bdist_dir,
-                supplied_platform=plat_name if self.plat_name_supplied else None)
-            # XXX switch to this alternate implementation for non-pure:
-            if not self.py_limited_api:
-                assert tag == supported_tags[0], "%s != %s" % (tag, supported_tags[0])
```

Comment:
Is this check important? The order of tags returned from `sys_tags` is different.

Reply:
Probably not, but I didn't write that code.